packages/client/src/objectSet/ObjectSetListenerWebsocket.ts

```diff
@@ -454,7 +454,11 @@ export class ObjectSetListenerWebsocket {
 
     for (const osdkObject of osdkObjectsWithReferenceUpdates) {
       if (osdkObject != null) {
-        sub.listener.onChange?.(osdkObject);
+        try {
+          sub.listener.onChange(osdkObject);
+        } catch (error) {
+          sub.listener.onError([error]);
```

Comment:
I'm going to FLUP this and add an enum of what caused the error. This would be (excuse the weird names) `SUBSCRIPTION_ERROR` or `HANDLER_ERROR`. This would make more sense if we could add a third that would suggest trying again, but this isn't something we can ensure